.github/workflows/release.yml

```diff
@@ -75,6 +79,16 @@ jobs:
                 adduser -u $(id -u) -G build -D -H build &&
                 su build -c 'cargo run -p package -- dist'
               "
+      - name: Package scie-jump ${{ needs.determine-tag.outputs.release-tag }} binary
+        if: ${{ matrix.target == 'armv7l' }}
+        run: |
+          docker run --rm --privileged multiarch/qemu-user-static --reset -p yes
+          docker build . \
+            -f Dockerfile.armv7l \
+            -t jump/armv7l \
+            --build-arg GROUP="$(id -g)" \
+            --build-arg USER="$(id -u)"
+          docker cp $(docker create jump/armv7l):/code/dist .
```

Comment:
One potential problem is that the final binary is called `scie-jump-linux-arm` and not `scie-jump-linux-armv7l` because this: https://github.com/a-scie/jump/blob/4f9d4122dbfbd21a48473d5b316da2d57710d575/package/src/main.rs#L184. See: https://doc.rust-lang.org/std/env/consts/constant.ARCH.html

Do you mind if I change the code to make this a special case?

Reply:
I did this now.